src/components/LandingPage/index.tsx

```diff
@@ -24,37 +24,43 @@ const LandingPage = (): JSX.Element => {
         <React.Fragment>
             <Content className={styles.content}>
                 <div className={styles.panel}>
-                    <h1>Simularium {BetaTag}</h1>
-                    <h2>
-                        Share, visualize, & interrogate biological simulations
-                        online
-                    </h2>
-                    <br />
-                    <p>
-                        The Simularium Viewer makes it easy to share and
-                        interrogate interactive 3D visualizations of biological
-                        simulation trajectories and related plots directly in a
-                        web browser. Its primary goal is to facilitate
-                        collaborations among experimental and computational
-                        biologists by removing major challenges to sharing,
-                        accessing, and comparing simulation results.
-                    </p>
+                    <div className={styles.textContent}>
```

Comment:
this looks like a lot of changes but all i did was add a div around the text content instead of styling with `.panel *` which I initially thought might be causing the problem, but left because it seems safer